debugging.md

```diff
@@ -877,3 +877,25 @@ You can also overlay such a grid on non-Material applications by using
 the
 [`GridPaper`](https://docs.flutter.io/flutter/widgets/GridPaper-class.html)
 widget directly.
+
+## Common problems
+
+{% comment %}
+Rewrite the following when we have a larger collection of problems.
+{% endcomment %}
+The following is a problem that some have encountered on MacOS.
+
+### "Too many open files" exception (MacOS)
+
+The Mac file system has a default limit on how many files it can have open
```

Comment:
My understanding is that this is not a problem with the file system, but rather a safety measure enforced by the shell (https://ss64.com/bash/ulimit.html). 

Also: Other operating system also have a default limit of open files as well. Their limits are just much higher by default so people are unlikely to run into problems.

I would maybe rephrase this to: 

```The default limit for Mac OS on how many files it can have open at a time is rather low. If you run into this limit, ...```